src/components/slider/slider.css

```diff
@@ -0,0 +1,102 @@
+.react-multi-carousel-list {
+    display: flex;
+    align-items: center;
+    overflow: hidden;
+    position: relative;
+}
+.react-multi-carousel-track {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    transform-style: preserve-3d;
+    backface-visibility: hidden;
+    will-change: transform, transition;
+}
+.react-multiple-carousel__arrow {
+    position: absolute;
+    outline: 0;
+    transition: all 0.5s;
+    border-radius: 35px;
+    z-index: 1000;
+    border: 0;
+    background: rgba(0, 0, 0, 0.5);
+    min-width: 35px;
+    min-height: 35px;
+    opacity: 1;
+    cursor: pointer;
+}
+.react-multiple-carousel__arrow:hover {
+    background: rgba(0, 0, 0, 0.8);
+}
+.react-multiple-carousel__arrow::before {
+    font-size: 25px;
+    color: #fff;
+    display: block;
+    text-align: center;
+    z-index: 2;
+    position: relative;
+}
+.react-multiple-carousel__arrow--left {
+    left: 10px;
+}
+.react-multiple-carousel__arrow--left::before {
+    content: '\21E6';
+}
+.react-multiple-carousel__arrow--right {
+    right: 10px;
+}
+.react-multiple-carousel__arrow--right::before {
+    content: '\21E8';
+}
+.react-multi-carousel-dot-list {
+    position: absolute;
+    bottom: 0;
+    display: flex;
+    left: 0;
+    right: 0;
+    justify-content: center;
+    margin: auto;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    text-align: center;
+}
+.react-multi-carousel-dot button {
+    display: inline-block;
+    width: 12px;
+    height: 12px;
+    border-radius: 50%;
+    opacity: 1;
+    padding: 5px 5px 5px 5px;
+    box-shadow: none;
+    transition: background 0.5s;
+    border-width: 2px;
+    border-style: solid;
+    border-color: grey;
+    padding: 0;
+    margin: 0;
+    margin-right: 6px;
```

Comment:
Suggest remove the line before and rewrite to margin: 0 6px 0 0;

Reply:
its default styles file from component! I Will try to resolve all comments